src/Tribe/Template.php

```diff
@@ -73,6 +73,15 @@ class Tribe__Template {
 	 */
 	protected $template_folder_lookup = false;
 
+	/**
+	 * Create a class variable for the include path, to avoid conflicting with extract.
+	 *
+	 * @since  4.11.0
+	 *
+	 * @var  string
+	 */
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 5d17f3f0f9c9853320e7d6ecd1a950fe</sub>